eos/constraints/B-to-P-form-factors.yaml

```diff
@@ -8,105 +8,86 @@
 # from [FNAL+MILC:2015A] (info on f_+ and f_0). Since the covariance matrices were singular, we
 # applied the modification suggested in 1901.02561 in a procedure explained in Section 4 of [LMvD:2021A]
 B->pi::f_++f_0+f_T@FNAL+MILC:2015C:
-  type: MultivariateGaussian(Covariance)
```

Comment:
I think we need a YAML linter pass by the Github Actions workflow and maybe as a pre-commit hook.

Reply:
I am not sure I know what you mean :)

Reply:
You spent time and effort to properly format and indent the YAML file. This can and should be automatized. New commits should be rejected if the YAML files are not following the formatting/indentation rules.